pkgs/tools/misc/trash-cli/default.nix

```diff
@@ -19,7 +17,8 @@ python3Packages.buildPythonApplication rec {
     (substituteAll {
       src = ./nix-paths.patch;
       df = "${coreutils}/bin/df";
-      libc = "${stdenv.cc.libc.out}/lib/libc.so.6";
+      libc = let ext = if stdenv.isDarwin then ".dylib" else ".so.6";
```

Comment:
I think you can replace this with: stdenv.hostPlatform.extensions.sharedLibrary. The .6 part should be optional.

Reply:
Thanks for suggestion. I replaced this line with
```
libc = "${stdenv.cc.libc}/lib/libc${stdenv.hostPlatform.extensions.sharedLibrary}";
```
and the build failed with
```
OSError: /nix/store/pr73kx0cdszbv9iw49g8dzi0nqxfjbx2-glibc-2.27/lib/libc.so: invalid ELF header
```
on Linux. `.6` seems to be necessary.

Reply:
Very weird never mind then!